### ISSUE TITLE
Two-factor authentication error

### ISSUE THREAD

**Issue by djpaoloc** [davidazevedo/InstagramLive-PHP#4]
when trying to authenticate the system response with this error:

$ php -f goLive.php 
Loading InstagramLive-PHP v0.3...
Logging into Instagram...
Code: xxxxxx
Error 1 While Logging in to Instagram: InstagramAPI\Response\LoginResponse: Challenge required.

**Comment by fathnojoum** [comment]
true, I also experienced it. Hopefully it will be fixed soon, because this is very useful. Thanks

**Comment by reinaldoteixeira** [comment]
I have the same error, can someone help me?

**Comment by socialAPIS** [comment]
Should i implement live in my api, and then change the import in composer.json.
When used in cmd my 2FA works.

Mgp25 is outdated and the lib uses deprecated login method.

**Comment by JR-Morton** [comment]
I was getting this exact same error and tried all kinds of methods of changing the code, but turns out (in my case) the error was being thrown as either "Challenge required" and would exit or the same two-factor error you all are seeing. I knew I didn't have two-factor authentication enabled on Instagram so what it means is that on my phone Instagram was noticing an unusual login attempt after each login. So what I did was enable two-factor authentication with SMS and it worked! Then I disabled two-factor authentication again and was no longer being stopped by this bug!
